src/nasher/init.nim

```diff
@@ -37,24 +37,24 @@ proc init*(opts: Options, pkg: PackageRef): bool =
 
   display("Initializing", "into " & dir)
 
-  try:
-    display("Creating", "package file at " & file)
-    createDir(dir)
-    writeFile(file, genPackageText(opts))
-    success("created package file")
-  except:
-    fatal("Could not create package file at " & file)
-
   # TODO: support hg
   if opts.getOrPut("vcs", "git") == "git":
     try:
       display("Initializing", "git repository")
       if gitInit(dir):
         gitIgnore(dir)
-      success("initialized git repository")
+        success("initialized git repository")
     except:
       error("Could not initialize git repository: " & getCurrentExceptionMsg())
 
+  try:
```

Comment:
Yup, I can change that back.  I swapped them earlier because I was using a git function in the package creation function, but I was considering removing that anyway and leaving it as an option instead of a branch always being added to the package and default target.